bootstrap_readme.txt

```diff
@@ -0,0 +1,24 @@
+The bootstrap.py file can be used to build wesnoth on Windows OS.
+
+Steps:
+
+1. Install python 2.7. The default path is recommended, but if you must change it, make sure there are no space in the path.
+2. Run the script by double-clicking it.
+3. Choose the number of cores (processors) you wish to build boost and wesnoth with. It is recommended to have at least 500-700mb free per core used.
```

Comment:
There's a point there. The prefix "m" means "milli", not "mega", and the symbol for bytes is "B", not "b". In other words, SI units are case sensitive. (I'm not quite certain whether "b" is actually bits or something else.)